### PR TITLE
Feature/mds 3749

### PR DESCRIPTION
**Pull request by mhuseinov**
- moved equipment from activities to its own
- added now_submission.application_equipment_xref table as general for equipment from VFCBC (the other ones like mech_trenching_equip_xref preserved for the data reference and consistency)
- updated equipment linkage on Core side
- updated transmogrify to handle new equipment

image
image

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/review/ReviewActivities.js [hunk 1]
```diff
@@ -201,6 +202,15 @@ export const ReviewActivities = (props) => {
           <WaterSupply isViewMode={props.isViewMode} />
         </FormSection>
       </ScrollContentWrapper>
+      <ScrollContentWrapper
+        id="equipment"
+        title="Equipment"
+        data={props.noticeOfWork.equipment}
+        isViewMode={props.isViewMode}
+      >
+        {/* <Equipment initialValues={props.noticeOfWork.equipment} isViewMode={props.isViewMode} /> */}
```
Comment (mhuseinov) [inline services/core-web/src/components/noticeOfWork/applications/review/ReviewActivities.js:211]: This will be removed in separate pr, thanks @mmumby

File: services/core-web/src/components/noticeOfWork/applications/review/ReviewActivities.js [hunk 2]
```diff
@@ -201,6 +202,15 @@ export const ReviewActivities = (props) => {
...
+        <Equipment isViewMode={props.isViewMode} />
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/review/ReviewActivities.js:212]: Discussed on a call  - I can remove this in my PR

File: services/core-api/app/api/now_applications/models/activity_equipment_xref.py [hunk 3]
```diff
@@ -11,7 +11,10 @@ class ActivityEquipmentXref(Base):
 
     activity_summary_id = db.Column(
         db.Integer, db.ForeignKey('activity_summary.activity_summary_id'), primary_key=True)
-    equipment_id = db.Column(db.Integer, db.ForeignKey('equipment.equipment_id'), primary_key=True)
+    now_application_id = db.Column(
```
Comment (exzizt) [inline services/core-api/app/api/now_applications/models/activity_equipment_xref.py:14]: Can you remove the "primary_key=True"?
